crates/bevy_animation/src/lib.rs

```diff
@@ -267,6 +318,12 @@ impl AnimationPlayer {
         self.animation.elapsed = elapsed;
         self
     }
+
+    /// Enable root motion on the currently playing animation
+    pub fn enable_root_motion(&mut self, root_motion: RootMotion) -> &mut Self {
```

Comment:
From my understanding this is meant to be used as a builder pattern, but it could also be used to set the root motion afterwards, no? Could it be renamed to `set_root_motion` and accept `Option<RootMotion>`, that way there is a way to disable it afterwards?